Update okteto-manifest to document the header (#1014)
okteto/docs: versioned_docs/version-1.30/reference/okteto-manifest.mdx

- `driver`: Specifies the backend to divert requests. Use `nginx` (or leave this field unspecified) to use the default Nginx driver.
- `namespace`: The namespace that holds the full shareable environment. When a request is directed to a service that wasn't created by the `deploy` command, Okteto will automatically direct it to the environment running on this namespace.

> When `divert` is enabled, Okteto injects the header `baggage.okteto-divert` when a request goes through the developer's ingress. Okteto uses this to route the request between services on your personal namespaces and the shared environment. 

Author: @codyjlandstrom @ifbyol I'd like to document the header now that this is available on 1.30.1. Once we agree on the text I'll copy it to the other version.

> When `divert` is enabled, Okteto injects the header `baggage.okteto-divert` when a request goes through the developer's ingress. Okteto uses this to route the request between services on your personal namespaces and the shared environment. 

Reviewer: ```suggestion
When `divert` is enabled, Okteto injects the header `baggage.okteto-divert` into every request that passes through your development ingress. This header allows Okteto to route the request intelligently between services running in your personal namespace and services running in a shared environment.
```

:::tip

> We recommend propagating the header to downstream service calls to enable Okteto to continue diverting requests. 

Reviewer: ```suggestion
To maintain request routing across service boundaries, we recommend propagating the `baggage.okteto-divert` header to all downstream service calls.
```

> The header can also be used to route messages to different queues and topics, redirect requests to services in other namespaces, dynamically select which database to read or write, etc. This GitHub repository includes several samples illustrating the different ways that you can leverage divert.

Author: ```suggestion
The header can also be used to route messages to different queues and topics, redirect requests to services in other namespaces, dynamically select which database to read or write, etc. [This GitHub repository](https://github.com/okteto-community/divert-showcase/) includes several samples illustrating how you can leverage divert.
```

> The header can also be used to route messages to different queues and topics, redirect requests to services in other namespaces, dynamically select which database to read or write, etc. [This GitHub repository](https://github.com/okteto-community/divert-showcase/) includes several samples illustrating how you can leverage divert.

Reviewer: ```suggestion
Beyond request routing, this header can also be used to:
- Route messages to different queues or topics
- Redirect requests to services in other namespaces
- Dynamically select the correct database instance for read/write operations

Check out [this GitHub repository](https://github.com/okteto-community/divert-showcase/) for practical examples of how to integrate Divert into your service communication flows.
```
